ds000117/.bidsignore

```diff
@@ -1 +1,238 @@
-*_FLASH.*
\ No newline at end of file
+run-1_echo-1_FLASH.json
```

Comment:
why don't you use wildcards

Reply:
+1

Reply:
Ok! This was done by Rik, do you know his user in GitHub?

Reply:
umm ... nopes. This one: https://github.com/RikHenson ?

Reply:
Yes he is! :) 

@RikHenson

Reply:
this has not yet been addressed